File: 1328/main.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(breakPalindrome("aba"))
}

func breakPalindrome(palindrome string) string {
	if len(palindrome) <= 1{
		return ""
	}

	s := []rune(palindrome)
	for i := 0; i < len(s)/2; i++ {
		if s[i] != 'a' {
			s[i] = 'a'
			return string(s)
		}
	}
	s[len(s)-1] = 'b'
	return string(s)
}

File: 1328/main.rs
struct Solution{}

impl Solution {
    pub fn break_palindrome(palindrome: String) -> String {
        let n = palindrome.len();
        if n == 1 {
            return "".to_string();
        }
        let mut data: Vec<char> = palindrome.chars().collect();
        for i in 0..(n/2) {
            if data[i] != 'a' {
                data[i] = 'a';
                return data.iter().collect();
            }
        }
        data[n-1] = 'b';
        return data.iter().collect();
    }
}

fn main() {
    println!("{:#?}", Solution::break_palindrome("abb".to_string()));
}